examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -312,8 +365,30 @@ angular.module('cloudberry.common', [])
               cloudberryService.timeResult = result.value[0];
               cloudberryService.mapResult = result.value[1];
               cloudberryService.hashTagResult = result.value[2];
-            } else { // this is the {key: "done"} message
-              console.log("receive :", result.value);
+            }
+            // When the query is executed completely
+            else  {
```

Comment:
we should use `else if` to specifically detect the `{key: "done"}` result, and use the `else` for the exception case.

Reply:
I have modified the conditions to reflect this.